sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/misc.scala

```diff
@@ -283,3 +283,21 @@ case class TypeOf(child: Expression) extends UnaryExpression {
 
   override protected def withNewChildInternal(newChild: Expression): TypeOf = copy(child = newChild)
 }
+
+// scalastyle:off line.size.limit
+@ExpressionDescription(
+  usage = """_FUNC_() - Return the current user who is running SparkContext for current query execution.""",
```

Comment:
As a SQL function, we need to be engine-agnostic and define the semantic without Spark.

Reply:
right